lib/mirage.ml

```diff
@@ -1327,24 +1327,23 @@ let configure_makefile ~target ~root ~name info =
   newline fmt;
   append fmt "LIBS   = %s" libraries;
   append fmt "PKGS   = %s" packages;
+  let default_tags =
+    "warn_error(+1..49),warn(A-4-41-44),debug,bin_annot,\
+     strict_sequence,principal"
```

Comment:
principal ? Really ? It doesn't play very well with disambiguation ...
Also, no safe string ?